Wrappers.h

```diff
@@ -1,71 +1,48 @@
 #pragma once
 
+#include "DishWasher.h"
+#include "DishWashingProcess.h"
+#include "DrumMachine.h"
+#include "ServiceStation.h"
+#include "Project.h"
+
```

Comment:
If you're not using anything from these files **in this header file**, `forward declare` here.
then `#include` in the CPP files.

DON'T INCLUDE WHAT YOU DON'T USE

- [ ] revise

Reply:
Ah ok. I thought the `#include` is generally in the header file. Thanks for the hint!